entrypoint.sh

```diff
@@ -260,7 +251,7 @@ test_go_ceph() {
         setup_mirroring
         export MIRROR_CONF
     fi
-    for pkg in "${pkgs[@]}"; do
+    for pkg in ${pkgs}; do
```

Comment:
Could I please bother you to keep pkgs as an array? I know that it isn't the most common thing in shell scripting, but if we're going to use bash I would like to retain the "safety" of knowing that if we were to have spaces in our dir names this could continue to work without change.

Reply:
These are package names, and the go spec doesn't allow spaces in them, so there is zero chance that could happen.

But out of curiosity, because I might need it some other say: how do you split multi-line string (the output of `go list`) into an array in bash?

Reply:
OK, its probably fine as it is then. If it becomes an issue in the future we can change it at a later time.

For a simple split like this you can do something like: `x=($(go list ./...))` and x will be an array. But IF the go list could contain spaces, etc you need a little more. There's some helpful hints here: https://stackoverflow.com/questions/9449417/how-do-i-assign-the-output-of-a-command-into-an-array